src/VisualStudio/Core/Def/Implementation/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -106,12 +105,9 @@ private async Task StartAsync(CancellationToken cancellationToken)
             }
         }
 
-        private async Task StartWorkerAsync(CancellationToken cancellationToken)
+        private async Task StartWorkerAsync()
         {
-            _workQueue = new AsyncBatchingWorkQueue<DesignerAttributeData>(
-                TimeSpan.FromSeconds(1),
-                this.NotifyProjectSystemAsync,
-                cancellationToken);
+            var cancellationToken = ThreadingContext.DisposalToken;
```

Comment:
Consider having StartWorkerAsync still take the cancellation token and then passing this in StartAsync(); I otherwise was scratching my head why StartAsync() was catching OperationCancelled exceptions when it wasn't passing a token in.